src/components/views/messages/DisambiguatedProfile.tsx

```diff
@@ -40,7 +40,7 @@ export default class DisambiguatedProfile extends React.Component<IProps> {
 
         let colorClass: string | undefined;
         if (colored) {
-            colorClass = getUserNameColorClass(fallbackName);
+            colorClass = getUserNameColorClass(mxid ?? "");
```

Comment:
Is there a test for this? I'd love to see one if not. Happy for you to merge and add it after if needed though.

Reply:
I will do that! thank you 🙏